integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomainModelInPV.java

```diff
@@ -134,19 +158,30 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
     assertNotNull(namespaces.get(0), "Namespace list is null");
     String opNamespace = namespaces.get(0);
 
-    // get a unique domain namespace
-    logger.info("Getting a unique namespace for WebLogic domain");
+    // get a unique domain1 namespace
+    logger.info("Getting a unique namespace for WebLogic domains");
     assertNotNull(namespaces.get(1), "Namespace list is null");
     domainNamespace = namespaces.get(1);
```

Comment:
WebLogic domain ( we have just one right?)

Reply:
no there are 2 domains created, its parameterized test